index.js

```diff
@@ -57,6 +57,15 @@ FileBin.prototype.write = function (fileName, data) {
   });
 };
 
+FileBin.prototype.destroy = function (fileName) {
+  return new RSVP.Promise((resolve, reject) => {
+    fs.unlink(path.join(this.base, fileName), (error) => {
+      if (error) { return reject(error); }
+      resolve(formatFile(fileName));
```

Comment:
It's kind of weird to return our now dead file. What about `resolve(true)` instead?